tests/test_units/test_initialize_case.py

```diff
@@ -145,7 +145,7 @@ def test_inicase_generate_case_metadata_exists_so_fails(
     casemetafolder = fmurun_w_casemetadata.parent.parent
 
     icase = InitializeCase(globalconfig2)
-    with pytest.warns(UserWarning, match=r"The metadata file already exist!"):
+    with pytest.warns(UserWarning, match=r"The case metadata file already exists"):
```

Comment:
Rehashed the warning text

Reply:
I wonder if it should rather say something like "Updating existing case metadata". As it stands now, I don't think we give the reader any hints of what he should actually do with this information. It's perhaps more for information, than warning - particularly when ERT has already informed the user about overwriting existing case.

?

Reply:
Something of a hint is given at least -- the full warning text is

> The case metadata file already exists and will not be overwritten. To make new case metadata delete the old case or run on a different runpath.

I tried to run off your comment on other channels that we don't want to overwrite the case metadata and if we do, the regular patterns of doing so with FMU should be followed.